Asterisk.2013/Asterisk.NET/Common.cs

```diff
@@ -26,7 +27,29 @@ public static class Common
         public static char INTERNAL_ACTION_ID_DELIMITER = '#';
 
         /// <summary> Variables delimiter </summary>
-        public static char[] VAR_DELIMITER = {'|'};
+        public static Dictionary<string, char[]> VAR_DELIMITERS = new Dictionary<string, char[]>();
+
+        /// <summary> Variables delimiter getter </summary>
+        public static char[] GET_VAR_DELIMITER(string hostname)
+        {
+            if (!VAR_DELIMITERS.ContainsKey(hostname))
+            {
+                VAR_DELIMITERS.Add(hostname, new char[] { '|' });

```

Comment:
Why are you adding something to the dict in a getter? Wouldn't it be better to test if the dict contains the key, return the value if so, and if the dict doens't contain the key return a default?